doc/content/htgr/pb67_cardinal/index.md

```diff
@@ -0,0 +1,120 @@
+
+### udf file style=font-size:125%
+
+The +.udf+ file utilizes the kernal functions we defined in +.oudf+, and provides the detailed configurations of the ralated nekRS simulations. 
+
+!listing /htgr/pb67_cardinal/pb67.udf language=cpp
+
+### par file style=font-size:125%
+
+The +.par+ file lists the necessary input parameters to nekRS simulations. 
+
+!listing /htgr/pb67_cardinal/pb67.par language=ini
+
+The +CUDA+ backend is enabled in ```[OCCA]``` block to allow nekRS running on GPU hardware. The specific filtering settings allow us to smooth out high frequencies. If you are planning on doing a DNS you should not include any filtering.
+Next, the parameters in ```[PRESSURE]``` block are selected to have a short time-to-solution. We are able to achieve this by adding a preconditioner and a smoother. In NekRS version 21.1 the combination of semg preconditioner with ```chebyshev+asm``` smoothers works well, but in later versions 22.0+, using the amgx preconditioner with ```chebyshev+jac``` smoothers is much faster.
+
+### usr file style=font-size:125%
+
+The +.usr+ is a legacy from Nek5000 code, and we still use to specify the boundary conditons in Nek CFD calculations. 
+Here we prescribe every element a boundaryID for both velocity and temperature. We are able to check the type of element using ```cbc(ifc,iel,1).eq.'TOP'``` and assign boundary types based on that. 
+In this case we have 5 types of elements: inlet, outlet, outer wall, pebble wall, and chamfer wall. Notice, for the chamfer we assign the wall condition for velocity, but the +E+ condition for temperature. 
+This results in flow moving around the chamfered area without an additional heat flux. The chamfer is necessary to prevent a mesh singularity between touching pebbles.
+
+!listing /htgr/pb67_cardinal/pb67.usr language=fortran start=subroutine usrdat2() end=subroutine usrdat3 
+
+
+## MOOSE setup
+
+As previously stated, we strongly reccommended completing the CHT tutorials on CARDINAL, which fully describe every aspect of the MOOSE setup. Here, we will only cover case specific necessities.
+
+In this case we need to create two files: +nek.i+ and +moose.i+ pertaining to the NekRS and MOOSE parameters respectively.
+
+Starting with the simpler +nek.i+ which contains the following
+
+!listing /htgr/pb67_cardinal/nek.i
+
+In the ```[MESH]``` block, MOOSE will create a copy of the Nek mesh at the given boundaries. In this case it is at the pebble surfaces. 
+In the ```[Problem]``` block we define the name of corresponding NekRS files. 
+Next, we want MOOSE to use the same time steps as Nek so we prescribe that in the ```[Executioner]``` block. 
+In the ```[Output]``` block we tell MOOSE to output an exodus file of the shallow Nek copy every 1000 time steps. This can be a helpful check at the beginnng of the simulation to make sure you are using the correct boundaries. 
```

Comment:
```suggestion
In the ```[Output]``` block we tell MOOSE to output an exodus file of the shallow Nek copy every 1,000 time steps. This can be a helpful check at the beginning of the simulation to make sure you are using the correct boundaries. 
```